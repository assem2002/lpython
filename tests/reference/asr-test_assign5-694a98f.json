{
    "basename": "asr-test_assign5-694a98f",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_assign5.py",
    "infile_hash": "41144111ae3dad56feed42b1b148334a8faef310fbbabde10f3af8df",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_assign5-694a98f.stderr",
    "stderr_hash": "29c2388315bcc4ce0815c7236615f9454b040871b3c7f17181f0ce29",
    "returncode": 2
}