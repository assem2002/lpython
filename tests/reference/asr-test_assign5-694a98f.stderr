semantic error: Type mismatch in assignment, the types must be compatible
 --> tests/errors/test_assign5.py:6:5
  |
6 |     x = y
  |     ^   ^ type mismatch ('list[list[i32]]' and 'list[list[str]]')


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
